backend/event/design.md

```diff
@@ -64,4 +147,32 @@ When the consumer start, then send a subscribe action to broker. The topic is co
 ## 4.6 Work flow
```

Comment:
1. 4.1 to 4.3 is simple description, i think a list is ok.
2. "4.4 How to start does" not describe how to start, remove it.
3. "Work flow" should be "Workflow".